Tasks/Maven/mavencodeanalysis.ts

```diff
@@ -157,6 +162,12 @@ function createAndUploadBuildSummary(enabledTools:string[], modules:ma.ModuleAna
         buildSummaryLines.push(summaryLine);
     });
 
+    // Add a double space and a final line with descriptive text, if there were any violations to be reported
+    if (totalViolationsInBuild > 0) {
```

Comment:
It looks neater if we only tag this message on when there are violations, especially since there may be a change to not upload any artifacts if there are no violations.